### ISSUE TITLE
curl export: wrap the URL in single-quotes

### ISSUE THREAD

**Issue by noamtamim** [a85/POSTMan-Chrome-Extension#1161]
Some characters in URLs have to be escaped for shells. In most cases, wrapping the entire URL with single quotes is good enough. 
Example:

```
GET https://nghttp2.org/httpbin/get?p1=v1&p2=v2
```

should be:

```
curl -X GET -H "Cache-Control: no-cache" 'https://nghttp2.org/httpbin/get?p1=v1&p2=v2'
```

Instead of

```
curl -X GET -H "Cache-Control: no-cache" https://nghttp2.org/httpbin/get?p1=v1&p2=v2
```

**Comment by patrickpilch** [comment]
+1
All the ampersands in the request URL ending up creating shell background jobs.

**Comment by abhijitkane** [comment]
We'll push a fix for this in the next release. Thanks for reporting this.

**Comment by noamtamim** [comment]
Seems fixed now (with Version: 3.0.2, Updated: July 1, 2015). I'm getting this for the same example:
    curl --request GET \
      --url 'https://nghttp2.org/httpbin/get?p1=v1&p2=v2'
